Outside target!
Target

FAIL Event Timing pointerenter. promise_test: Unhandled rejection with value: object "TypeError: null is not an object (evaluating 'performance.eventCounts.get')"

